app/Http/Controllers/Three/SignupsController.php

```diff
@@ -53,4 +56,17 @@ public function show(Request $request, $id)
 
         return $this->item($signup, 200, [], $this->transformer, $request->query('include'));
     }
+
+    /**
+     * Delete a signup.
+     *
+     * @param  Object $signup
+     * @return \Illuminate\Http\Response
+     */
+    public function destroy(Signup $signup)
+    {
+        $signup->delete();
```

Comment:
@DFurnes I followed [this tutorial](https://laracasts.com/series/laravel-from-scratch-2017/episodes/9) and I think did everything needed for Route Model Binding however `$signup` here is just an empty signup model, so it's not actually deleting the record. Since we're on 5.4, I think that's all that is needed (in addition to the change made in the `routes` file but not sure why it's not working. Is there something I'm missing

Reply:
Hmmm weird! Let's look at this together IRL tomorrow morning.

Reply:
@DFurnes ok! and thanks for that @sbsmith86 ! I saw that somewhere too to add that but that didn't seem to work either :-/

Reply:
@sbsmith86 just in case you were wondering, the reason it wasn't working is because we weren't applying the `api` middleware to these routes which is where we were calling `\Illuminate\Routing\Middleware\SubstituteBindings::class,`. The PR is now updated to use the `api` middleware!